### PR TITLE
Add Checklist explaination to the doc

### PR DESCRIPTION
**Pull request by 4rnoP**
* Resolves: #1456 
* Target version: master 

### Summary
Add checklist explaination to the Markdown doc

### CODE REVIEW

File: docs/Markdown.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ That same link offers also a comprehensive list of what is supported, and what i
 
 [CommonMark Markdown Reference](http://commonmark.org/help/)
 
+## Note about checklists
+
+It is possible to create checklist in Deck by writing it in Markdown, using the following syntax:
```
Comment (stefan-niedermann) [inline docs/Markdown.md:19]: `to create a checklist` or `to create checklists`, no?

File: docs/Markdown.md [hunk 2]
```diff
@@ -14,6 +14,16 @@ That same link offers also a comprehensive list of what is supported, and what i
...
+```md
+- [ ] This is a unchecked item
```
Comment (stefan-niedermann) [inline docs/Markdown.md:21]: `... an unchecked...`

File: docs/Markdown.md [hunk 3]
```diff
@@ -14,6 +14,16 @@ That same link offers also a comprehensive list of what is supported, and what i
...
+- [x] This is a checked item
+```
+Then, the items can be checked and unchecked by clicking on the rendered checkbox.
+Also, a summary of the completed items will be visible under the card element.
```
Comment (stefan-niedermann) [inline docs/Markdown.md:25]: `...visible below the...` or maybe better `...visible at the bottom of the...`?

**Comment by juliushaertl** [comment]
Thanks a lot :)

**Pull request merged**